src/10_functions.py

```diff
@@ -1,12 +1,18 @@
 # Write a function is_even that will return true if the passed-in number is even.
-
 # YOUR CODE HERE
 
+
+def isEven(n):
+    return not n % 2
```

Comment:
This can possibly be confusing because the function is called `is_even` but when you're reading the return, its saying `not n%2`.

Not wrong by any means but had to take a double glance at what you were writing. 😆

Reply:
Heyyyyy I'm just keeping you on your toes